cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -233,7 +233,7 @@ def test_notifications_handler_dismiss(self):
             # delete nofications that are dismissed
             CourseRerunState.objects.get(id=rerun_state.id)
 
-        self.assertTrue(has_course_author_access(user2, rerun_course_key))
+        self.assertFalse(has_course_author_access(user2, rerun_course_key))
```

Comment:
@tobz: The reason why `user2` is not supposed to have course author access here is because this is testing the case where creating the course rerun fails. When that happens, we're supposed to call `_dismiss_notification()`, which in turn calls `remove_all_instructors()`, which is supposed to remove all staff from that (broken) course rerun.